Alicerce/Sources/Extensions/UICollectionView+ReusableView.swift

```diff
@@ -19,11 +19,12 @@ public extension UICollectionView {
         return cell
     }
     
-    public func register<T: UICollectionViewCell>(_ cellType: T.Type) where T: ViewCellProtocol {
+    public func register<T: UICollectionViewCell>(_ cellType: T.Type) where T: ReusableView {
```

Comment:
I don't think we need to specify `public` for all inner vars and methods, since we specify it at a higher scope level. More cases throughout the PR 😉